spec/integration/messages/i18n_spec.rb

```diff
@@ -12,21 +11,24 @@
 
   before do
     I18n.config.available_locales = [:en, :pl]
-    I18n.load_path.concat(%w(en pl).map { |l| SPEC_ROOT.join("fixtures/locales/#{l}.yml") })
+    I18n.load_path.concat(%w[en pl].map { |l| SPEC_ROOT.join("fixtures/locales/#{l}.yml") })
     I18n.backend.load_translations
     I18n.reload!
   end
 
-  describe '#[]' do
+  describe '#lookup' do
```

Comment:
@robhanlon22 why did you have to change the spec?

Reply:
`#[]`/`#call` returned instances of `Template`, but `Template` was removed in this PR, so `#[]`/`#call` was removed, thus the change to the spec